@carias8 @meteor-man hmm so I get the following when I try to hit that endpoint. Can you advise?
```
127.0.0.1 - - [29/Nov/2016 09:57:36] "POST /clients?name=test HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/phil/anaconda/envs/py2/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/phil/anaconda/envs/py2/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/phil/anaconda/envs/py2/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/phil/anaconda/envs/py2/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/phil/anaconda/envs/py2/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/phil/anaconda/envs/py2/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/phil/anaconda/envs/py2/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/phil/anaconda/envs/py2/lib/python2.7/site-packages/flask/app.py", line 1617, in dispatch_request
    self.raise_routing_exception(req)
  File "/home/phil/anaconda/envs/py2/lib/python2.7/site-packages/flask/app.py", line 1603, in raise_routing_exception
    raise FormDataRoutingRedirect(request)
FormDataRoutingRedirect: A request was sent to this URL (http://127.0.0.1:5000/clients?name=test) but a redirect was issued automatical
ly by the routing system to "http://127.0.0.1:5000/clients/?name=test".  The URL was defined with a trailing slash so Flask will automa
tically redirect to the URL with the trailing slash if it was accessed without one.  Make sure to directly send your POST-request to th
is URL since we can't make browsers or HTTP clients redirect with form data reliably or without user interaction.

```

--- a/chloroform/routes.py
+++ b/chloroform/routes.py
@@ -49,7 +49,7 @@ def show(model_name, model_id,title):
 @app.route('/<string:model_name>/', methods=['POST'])
 def create(model_name):
     model = get_model_from_string(model_name)
-    instance = model(**request.form.to_dict())
+    instance = model(**request.json)
     db.session.add(instance)
     db.session.commit()
     return "success\n"
@@ -59,7 +59,7 @@ def create(model_name):
 def update(model_name, model_id):
     model = get_model_from_string(model_name)
     instance = model.query.filter(model.id == model_id)
-    instance.update(request.form)
+    instance.update(request.json)
     db.session.commit()
     return "success\n"
 

--- a/chloroform/routes.py
+++ b/chloroform/routes.py
@@ -46,7 +46,7 @@ def show(model_name, model_id,title):
         return dump_to_json(rows)
 
 # curl --data "name=new_client" localhost:5000/clients
-@app.route('/<string:model_name>', methods=['POST'])
+@app.route('/<string:model_name>/', methods=['POST'])
 def create(model_name):
     model = get_model_from_string(model_name)
     instance = model(**request.form.to_dict())

--- a/chloroform/static/app/controllers.js
+++ b/chloroform/static/app/controllers.js
@@ -13,11 +13,11 @@ mainApp.controller('ClientCtrl', [ '$scope', '$http', 'searchService',
 			};
             $scope.showClientNew = false;
             $scope.clientNew= null;
-            $scope.createModel = function(model) {
-                return $http.post('/clients/?name=', '', {
-                    params : {name : model}
+            $scope.createModel = function(name) {
+                return $http.post('/clients/', {
+                    name: name
                 }).then(function(response){
-                        console.log(response);
+                    console.log(response);
                     return response.data.map(function(item){
                         return item;
                     });

--- a/chloroform/static/app/controllers.js
+++ b/chloroform/static/app/controllers.js
@@ -14,7 +14,7 @@ mainApp.controller('ClientCtrl', [ '$scope', '$http', 'searchService',
             $scope.showClientNew = false;
             $scope.clientNew= null;
             $scope.createModel = function(model) {
-                return $http.post('/clients/'+model, '', {
+                return $http.post('/clients?name=' + model, '', {
                     params : {}
                 }).then(function(response){
                         console.log(response);

--- a/chloroform/static/app/controllers.js
+++ b/chloroform/static/app/controllers.js
@@ -14,8 +14,8 @@ mainApp.controller('ClientCtrl', [ '$scope', '$http', 'searchService',
             $scope.showClientNew = false;
             $scope.clientNew= null;
             $scope.createModel = function(model) {
-                return $http.post('/clients?name=' + model, '', {
-                    params : {}
+                return $http.post('/clients/?name=', '', {
+                    params : {name : model}
                 }).then(function(response){
                         console.log(response);
                     return response.data.map(function(item){

